leakcanary-android/src/main/java/com/squareup/leakcanary/internal/DisplayLeakActivity.java

```diff
@@ -364,9 +376,29 @@ public void onItemClick(AdapterView<?> parent, View view, int position, long id)
         });
       }
       actionButton.setVisibility(leaks.size() == 0 ? GONE : VISIBLE);
+      shareButton.setVisibility(GONE);
     }
   }
 
+  private void shareLeakToStackOverflow() {
+    AnalyzedHeap visibleLeak = getVisibleLeak();
+    final String leakInfo = LeakCanary.INSTANCE.leakInfo(this, visibleLeak.heapDump, visibleLeak.result, false);
+    final Toast toast = Toast.makeText(this, R.string.leak_canary_leak_copied, Toast.LENGTH_SHORT);
```

Comment:
LENGTH_SHORT => LENGTH_LONG? People are going to miss this. Another option might be a dialog that explains what's up but that could be annoying.